components/addTask.vue

```diff
@@ -40,22 +43,79 @@
       Add Task
     </button>
   </aside>
+   <div class="mx-auto flex justify-center mt-10 px-4">
+    <span class="inline-block justify-between bg-blue-600 py-1 mb-5 px-9 text-sm text-white font-bold rounded-full ">Available Tasks</span>
+  </div>
+ 
+
+  </main>
 </template>
 
 <script>
-import { defineComponent } from '@nuxtjs/composition-api'
+import {
+  defineComponent,
+  reactive,
+  toRefs,
+  useContext,
+} from '@nuxtjs/composition-api'
 
 export default defineComponent({
   emits: ['newTask'],
-  methods: {
-    addTask() {
+  
+  setup(props,context){
+   const state = reactive({
+      addtitle:'',
+    })
+     const {$axios,store,$toast} = useContext()
+
+  
+    function addTask() {
       /**
        * @todo Complete this function.
        * @todo 1. Send the request to add the task to the backend server.
        * @todo 2. Add the task in the dom.
        * @hint use emit to make a event that parent can observe
        */
-    },
+      
+        // console.log(state.addtitle);
+    if (!state.addtitle) {
+        return;
+    }
+    const headers={
+       Authorization: 'Token ' + store.getters.token,
+    }
+    const data={
+      title: state.addtitle,
+    }
+    $axios.post('todo/create/',data,{headers})
+       
+        .then(function () {
+           context.emit('newTask'),
+           state.addtitle='',
+             $toast.success('Task Added Successfully')
+
+
+        })
+        .catch(function (err) {
+             $toast.error(
+            'Something went wrong, try again'
+          )
+        });
+   
+  }
+  return{
+      addTask,
+  ...toRefs(state)
+  
+  }
   },
 })
 </script>
+<style  scoped>
+#addbtn{
+  background-color: lightgreen;
+}
+#addbtn:hover{
+  border: 0.5px solid rgb(23, 172, 23);
+}
+</style>
```

Comment:
just an early review you can do all this with windicss classes only

Reply:
ok bhaiya thanx, will sure look at this issue.